processing/src/test/java/io/druid/query/groupby/GroupByQueryRunnerTest.java

```diff
@@ -831,7 +831,7 @@ public String apply(String dimValue)
         .build();
 
     List<Row> expectedResults = Arrays.asList(
-        GroupByQueryRunnerTestHelper.createExpectedRow("2011-04-01", "alias", "", "rows", 3L, "idx", 2870L),
+        GroupByQueryRunnerTestHelper.createExpectedRow("2011-04-01", "alias", null, "rows", 3L, "idx", 2870L),
```

Comment:
@navis I think we should be consistent in returning null or empty string to clients. I believe right now Druid will always try to return empty string. Can we do a null to empty string conversion within the groupBy code?